ceaos/grow.py

```diff
@@ -0,0 +1,150 @@
+from ceaos.actuators.actuator_commands import set_airtemp
+import yaml
+from datetime import datetime, time
+from .objects.farm import Farm
+from .objects.beds import Bed
+from .objects.environment import Environment
+from .actuators import actuator_commands
+from importlib_resources import files
+
+
+def get_allbeds(farm):
+    all_beds = dict()
+    environments = farm.get_envs()
+    for environment in environments:
+        beds = environments[environment].get_beds()
+        for bed in beds:
+            all_beds[beds[bed].get_name()] = beds[bed]
+
+    return all_beds
+
+
+def find_relevantbeds(bed_names_list, all_beds):
+    bed_list = []
+    for names in bed_names_list:
+        for names2 in all_beds:
+            if names == names2:
+                bed_list.append(all_beds[names2])
+
+    return bed_list
+
+
+def find_ips(bed_list, actuator):
+    ip_list = []
+    for bed in bed_list:
+        for act in bed.get_actuators():
+            if actuator in bed.get_actuators()[act]:
+                ip_list.append(act)
+
+    return ip_list
+
+
+def send_command(ip_list, max, min, actuation):
+    for ip in ip_list:
```

Comment:
This loop will send a command to every IP in the list. Sounds a bit much, right?